### PR TITLE
Python3 Compatibility: added parentheses to prints

### PR DESCRIPTION
**Pull request by stasiek**
Tests return same results for Python 2 and 3.

### CODE REVIEW

**Comment by pekkaklarck** [comment]
Python 3 compatibility is a great idea. I submitted separate issue #2 about it with some additional tasks related to it.

Regarding to this PR, it's great to see that needed changes are minimal. I'd add `from __future__ import print_function` to affected files, though. Using `print('foo')` in Python 2 without it works in simple cases, but I don't think that's a good approach except for one-liners and very simple scripts.

**Comment by stasiek** [comment]
@pekkaklarck got no time to think about what else to change. For now I just need it for SSHLibrary Python 3 compatibility.
Changes required in #2 included, I've also updated the copyright note.

**Comment by stasiek** [comment]
Forgot about importing __future__.print_function. I'll add it later today.

File: README.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Introduction
 StatusChecker is a tool for validating that executed `Robot Framework`_
 test cases have expected statuses and log messages. It is mainly useful
 for Robot Framework test library developers who want to use Robot
-Framework to also test their libraries.
+Framework to also test their libraries. Works with Pythons 2 and 3.
```
Comment (pekkaklarck) [inline README.rst]: add `since version 1.3` at the end here.

**Comment by pekkaklarck** [comment]
Cool, let's just create 1.3 with Python 3 compatibility then. Can you add `print_function` import and mention version that added Python 3 support in README (see my line note)? I can then create the release.

**Comment by stasiek** [comment]
Should be ready to merge now.

File: robotstatuschecker.py [hunk 2]
```diff
@@ -54,17 +54,19 @@
 from robot.api import ExecutionResult, ResultVisitor
 from robot.utils import Matcher
 
+from __future__ import print_function
```
Comment (pekkaklarck) [inline robotstatuschecker.py]: Future imports should be before other imports and code. See https://docs.python.org/2/reference/simple_stmts.html#future

**Comment by pekkaklarck** [comment]
Two issue related to future imports:

- They should be before other code and imports. See https://docs.python.org/2/reference/simple_stmts.html#future

- Should also add the import to `run.py`.

Do you @stasiek have time to fix these today or should  I do it myself? If you can fix them ASAP, I ought to still have time for creating a release. I have some other task that I need to do first, though.

**Pull request merged**